system-int-test/src/cmr/system_int_test/utils/bootstrap_util.clj

```diff
@@ -18,108 +18,133 @@
 
 (defn bulk-index-after-date-time
   "Call the bootstrap app to bulk index concepts with revision dates later than the given datetime."
-  [date-time]
-  (let [response (client/request
-                   {:method :post
-                    :query-params {:synchronous true}
-                    :url (url/bulk-index-after-date-time-url date-time)
-                    :content-type :json
-                    :accept :json
-                    :throw-exceptions false
-                    :connection-manager (s/conn-mgr)})
-        body (json/decode (:body response) true)]
-    (assoc body :status (:status response))))
+  ([date-time]
+    (bulk-index-after-date-time date-time {transmit-config/token-header (transmit-config/echo-system-token)}))
```

Comment:
This line has one extra space - should line up directly below the `[`. Same for the other multiple arrity methods.

Reply:
Fixed.